spec.html

```diff
@@ -13713,7 +13713,7 @@ <h1>
           1. Let _newLenDesc_ be a copy of _Desc_.
           1. [id="step-arraysetlength-newlen"] Let _newLen_ be ? ToUint32(_Desc_.[[Value]]).
           1. [id="step-arraysetlength-numberlen"] Let _numberLen_ be ? ToNumber(_Desc_.[[Value]]).
-          1. If _newLen_ is not the same value as _numberLen_, throw a *RangeError* exception.
+          1. If _newLen_ &ne; _numberLen_, throw a *RangeError* exception.
```

Comment:
Thanks for the patch! I agree this is incorrect currently. However, I would prefer that we avoid the use of `=` (or `≠`) to compare Number values, particularly when one of them might be `-0`, because [it's ambiguous](https://github.com/tc39/ecma262/pull/2007/#issuecomment-637070282) in cases like this. ("is the same value as" is unambiguous, but here it's incorrect.)

Instead, we can use a comparison operation that makes it explicit that the floats `-0` and `0` should be considered to equivalent, as in

```suggestion
          1. If SameValueZero(_newLen_, _numberLen_) is *false*, throw a *RangeError* exception.
```

Reply:
I understand, and I agree with your suggestion! Thank you.